apps/web-giddh/src/app/shared/header/components/new-group-account-sidebar/groups-account-sidebar.component.scss

```diff
@@ -1,9 +1,8 @@
 @import "../../../../../assets/css/color.scss";
-
 .list-item:focus,
 .list-item:focus .list-item {
     color: $white !important;
-    background: $primary;
```

Comment:
@aditi-tiwari17 @shalinee-walkover Please check this code revert

Reply:
checked, its fine